helpers/dom.js

```diff
@@ -227,3 +227,19 @@ export function isVisible(node) {
 	return true;
 
 }
+
+export function querySelectorComposed(node, selector) {
+
+	const elem = node.querySelector(selector);
```

Comment:
One note: this will fail if `node` is undefined, null, or otherwise not a node type. Just wondering if you considered doing a check and throwing a more friendly error - but I think it's fine as-is.

Reply:
Good call -- added type checking for both parameters.